### DISCUSSION TITLE
Server Restart / Script Option or host commands

### DISCUSSION THREAD

**Discussion by robbudge** [frangoteam/FUXA#1059] [Ideas]
The common solution  to most problems is a restart this is normally is an easy task.
With Fuxa running as Client / Server.
Restarting the client  session is a simple refresh  command that can be added to a button.
Getting the server to restart like issuing 'PM2 restart main' on the host is a little more problematic.
could a shell command be added to the script to allow any number of host commands to be  sent.
Or at least a single command for a server reboot / fuxa restart.

My Concern is simply how could the end user from the client, force a restart of the fuxa  server , just as a backup

**Comment by mariorl** [comment]
Hi, not sure if it fits for you, but you can kill the whole server runtime within an script and then detect it externally using an external script and the file system, and trig a re-start.
